core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -996,7 +997,9 @@ public void format() throws IOException {
       if (journalPath.exists()) {
         FileUtils.delete(journalPath.getAbsolutePath());
       }
-      journalPath.mkdirs();
+      if (journalPath.mkdirs()) {
```

Comment:
I don't understand this.
Throw if creation of folders succeed?

Reply:
I'm so sorry.
I checkout a new branch, forget '!', and also not test it again.